mgmtd/mgmt_main.c

```diff
@@ -196,6 +197,8 @@ static const struct frr_yang_module_info *const mgmt_yang_modules[] = {
 	&ietf_key_chain_cli_info,
 	&ietf_key_chain_deviation_info,
 
+	&ietf_srv6_types_info,
+
```

Comment:
one single line is necessary

Reply:
Can you please clarify this a bit?

Reply:
this is a bit "extreme". we spare the number of chars we push to git :). so an extra line is unnecessary.